EventKit/eventkit-cloud
# Adds option to sort by favorites

# Description of Improvement

Adds option to sort by favorites in create datapack workflow.

## How to test

Go to Create Datapack
Set some favorite providers
Apply sort by favorite
Verify product list is now sorted by favorites
Profit

## Quality Assurance

The following items have been updated:


## Screenshots

<img width="1786" alt="Screen Shot 2022-09-02 at 3 47 31 PM" src="https://user-images.githubusercontent.com/3164631/188226295-14423575-a9f5-4261-a9d4-f8d539c00b4f.png">

:smile:
